sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -700,20 +701,33 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
 
   test("create temporary view using") {
     val csvFile =
-      Thread.currentThread().getContextClassLoader.getResource("test-data/cars.csv").toString
-    withView("testview") {
-      sql(s"CREATE OR REPLACE TEMPORARY VIEW testview (c1 String, c2 String)  USING " +
-        "org.apache.spark.sql.execution.datasources.csv.CSVFileFormat  " +
-        s"OPTIONS (PATH '$csvFile')")
+      Thread.currentThread().getContextClassLoader.getResourceAsStream("test-data/cars.csv")
```

Comment:
the point is that the return value of `getResource.toString` is 'jar:file:/xxx.jar!/test-data/cars.csv'.If we use this path to set table location as following code in the test case, this path will cause exception when it is used to construct `new Path()` in [here](https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSource.scala#L344).

So I copy this file from resource in jar to a temp file, which path is 'file:/tmppath/cars.csv', and use this normal path to set table location ,it is ok for the following code.